### DISCUSSION TITLE
Improve get_product

### DISCUSSION THREAD

**Discussion by barbax7** [sergioteula/python-amazon-paapi#57] [Ideas]
In `get_product` function there is a `split(",")` method that split passed string and if the returned list's lenght is greater than one, it raises an error.

I think it could be better that with `get_product` function you get only the product of first asin (or url), ignoring the rest.

It could be useful also when you have urls like `https://www.amazon.it/canzone-Achille-Madeline-Miller-ebook/dp/B07LGBM18M?__mk_it_IT=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=R96BEWN0UBFE&dchild=1&keywords=la+canzone+di+achille&qid=1615931295&sprefix=la+canzone,aps,190&sr=8-2&linkCode=sl1&tag=flow074-21&linkId=6f4cfda5b4222977473aa183b503e2db&language=it_IT&ref_=as_li_ss_tl` where there are two commas inside.

What do you think?

**Comment by sergioteula** [comment]
Yes, I think it's a good idea, we can add this on a future update. Thanks for your suggestion!

**Comment by sergioteula** [comment]
This should be solved with the last version, since the `get_product` method disappeared and only `get_items` method is available.
